### ISSUE TITLE
Can you please verify this fix in tomorrow's pre-release edition of GitLens? Thank you so much!

### ISSUE THREAD

**Issue by hoangtuan438** [gitkraken/vscode-gitlens#3324]
Can you please verify this fix in tomorrow's pre-release edition of GitLens? Thank you so much!

You can switch to the pre-release edition of GitLens, by clicking on the "Switch to Pre-Release version of this extension" from the Extensions view.

![image](https://user-images.githubusercontent.com/641685/199397587-6948b1bc-3019-4d85-a6d5-332fea5d8f78.png)

_Originally posted by @axosoft-ramint in https://github.com/gitkraken/vscode-gitlens/issues/2409#issuecomment-1379256162_

Hi @axosoft,
I try to search commit message in my current local branch. But it always return all commits of unrelated branches. I don't see any options to select branch to search commit
![image](https://user-images.githubusercontent.com/20530338/334650865-a099200a-6287-4c64-b20a-d2225ee13691.png)

can you help me with that

**Comment by eamodio** [comment]
Yeah, we don't currently support filtering only to the current branch.

You can mostly get this to work using the Commit Graph -- if you filter it to the current branch and then enter your search query in the search box, then the results will only show for the current branch.

**Comment by hoangtuan438** [comment]
thank you for your response. It's on Pro version. Let me try it
